lib/paperclip/affine_clamp.rb

```diff
@@ -0,0 +1,156 @@
+module Paperclip
+  class AffineClamp < Paperclip::Processor
```

Comment:
It'd be cool to have some comments about what exactly you customized(what functions/blocks), just to leave some evidence for future reference, maybe some paperclip upgrades require some adjusting

Reply:
done https://github.com/geminimvp/solidus_product_feed/pull/11/files#diff-06ed3cfc6908c699efa17b5b571aa446R3